src/main/java/com/zoo/animals/Bird.java

```diff
@@ -1,4 +1,21 @@
 package com.zoo.animals;
 
-public class Bird {
-}
+public class Bird extends Animal {
+
+    private String wingSpan;
+
+    public Bird(int energyLevel, String name, int age, int weight) {
+        super(energyLevel, name, age, weight);
+    }
+
+    @Override
+    public void makeSound() {
+        System.out.println("Пташка щебече");
+    }
+
+    public void fly() {
+        int currentEnergy = getEnergyLevel();
```

Comment:
Not bad, here the usage of an additional variable is justified as it adds more clarity to what's happening and improves readability IMHO (even though someone can call it excessive)  👍